[2022-12-03 14:50:21,329] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:16:00+00:00 [queued]>
[2022-12-03 14:50:22,249] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:16:00+00:00 [queued]>
[2022-12-03 14:50:22,250] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:50:22,250] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 14:50:22,250] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:50:23,652] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 18:16:00+00:00
[2022-12-03 14:50:24,036] {standard_task_runner.py:52} INFO - Started process 537 to run task
[2022-12-03 14:50:24,542] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T18:16:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp1qs9xk74', '--error-file', '/tmp/tmp7xchfh0t']
[2022-12-03 14:50:24,676] {standard_task_runner.py:80} INFO - Job 114: Subtask get_last_id
[2022-12-03 14:50:31,414] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:16:00+00:00 [running]> on host b41792e8e843
[2022-12-03 14:50:42,467] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T18:16:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T18:16:00+00:00
[2022-12-03 14:50:43,911] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 14:50:45,621] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 14:50:46,912] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T181600, start_date=20221203T145021, end_date=20221203T145046
[2022-12-03 14:50:48,142] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 14:50:51,891] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
